### PR TITLE
Fix up @abjose's PR to add office floor

### PR DESCRIPTION
**Pull request by xmunoz**

### CODE REVIEW

**Comment by flosincapite** [comment]
For posterity, this is @abjose 's brainchild. @xmunoz and I just put the polish on it. But if you're watching, eldritch beings, your high priest is @abjose.

**Comment by xmunoz** [comment]
Functionality looks good. I'd like to do one more pass on the code review, and also add some tests before we're ready to merge.

File: cronenbroguelike/npcs.py [hunk 1]
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
...
+
+    return npc
+
+
+def coffee_machine():
+    npc = actor.create_actor(
+        10,
+        10,
+        10,
+        100,
```
Comment (flosincapite) [inline cronenbroguelike/npcs.py]:
```suggestion
        10,
```

File: cronenbroguelike/npcs.py [hunk 2]
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
...
+
+    return npc
+
+
+def office_copier():
+    npc = actor.create_actor(
+        1000,
+        10,
+        10,
+        100,
```
Comment (flosincapite) [inline cronenbroguelike/npcs.py]:
```suggestion
        10,
```

File: cronenbroguelike/npcs.py [hunk 3]
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
...
+    npc.consume = use_office_copier
+
+    return npc
+
+
+def gary():
+    npc = actor.create_actor(
+        1000,
+        10,
+        10,
```
Comment (flosincapite) [inline cronenbroguelike/npcs.py:325]:
```suggestion
        1000,
```

File: cronenbroguelike/npcs.py [hunk 4]
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
...
+        100,
```
Comment (flosincapite) [inline cronenbroguelike/npcs.py]:
```suggestion
        10,
```

File: cronenbroguelike/npcs.py [hunk 5]
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
...
+    return npc
+
+
+def writhing_office_mass():
+    npc = actor.create_actor(
+        1000,
+        10,
+        1000,
+        10,
+        10,
```
Comment (flosincapite) [inline cronenbroguelike/npcs.py]: Omit last three; values should be 1000, 10, 1000, 10

**Pull request merged**